src/components/AccordionSponsor/index.js

```diff
@@ -34,12 +23,11 @@ const AccordionSponsor = (props) => {
       <summary>
         <StyledAccordionHeader $isFilled={filled}>
           <StyledAccordionImageWrapper $bgImage={bgImage}>
-            <Frame color={frame} />
-            <StyledAccordionImage $rotate={imageRotatation}>
+            <StyledAccordionImage>
               <img {...image} />
             </StyledAccordionImage>
           </StyledAccordionImageWrapper>
-          <ContentMaster size="md" title={title}>
+          <ContentMaster size="md" title={<>{title}</>} hasCaret={false}>
```

Comment:
The original logic was if the text was a string it would do the typing animation and if it wasn't then it would render just text. This is just a workaround without adding additional props to prevent it from animating. I could rework it but  didn't feel like it was worth spending the additional time on. Let me know if you think otherwise.

Reply:
Ah cool, that makes sense and was what I suspected was happening?